### ISSUE TITLE
FR: Select a TV/field to grab "extract" from (instead of content)

### ISSUE THREAD

**Issue by mindeffects** [modxcms/SimpleSearch#7]
Hi there,
the search inside of TVs (including MIGX-TVs) is working fine, but the "extract" is allways coming from the content field, hence, the search result has only a "title" but no "extract" when the target is inside a TV.
I found a way to change that, but my solution depends on "not using the content field": By rendering a "fake" content into the "real" content field via a plugin at "OnDocSave", I can now search for everything I want, just inside the content field and get a nice extract.
Since the content field is sometimes needed for other things (like e.g. the content ;-) ), I would love to have a choice for the "source of the extract": I would like to choose a TV instead of the content field.

Long story short: I would love to set the "source of the extract" to any field or TV I wish.

Thanks a lot!
Oliver
